web/src/lottie-player.ts

```diff
@@ -252,6 +252,15 @@ export class LottiePlayer extends LitElement {
   @property({ type: Number })
   public currentState: PlayerState = PlayerState.Loading;
 
+  /**
+   * current resolution of animation (readonly)
+   * @since 1.0
+   */
+  @property({ type: Float32Array })
+  public get resolution(): Float32Array {
```

Comment:
1. Is it really necessary? If so,
2. I think it requires a consistent naming convention, such as 'size()' since the input function is 'resize()'.

Reply:
> 1. Is it really necessary? If so,
think so, Viewer needs to show original size (not canvas size)

<img width="316" alt="Screenshot 2024-01-04 at 4 39 55 PM" src="https://github.com/thorvg/thorvg/assets/11167117/63a31cd8-a9db-4c81-ab51-449d05ed27e1">

Reply:
> 2. I think it requires a consistent naming convention, such as 'size()' since the input function is 'resize()'

gotcha, will change to `size()`